flash/image/classification/transforms.py

```diff
@@ -91,3 +93,30 @@ def train_default_transforms(image_size: Tuple[int, int]) -> Dict[str, Callable]
     else:
         transforms = {"per_sample_transform": ApplyToKeys(DataKeys.INPUT, T.RandomHorizontalFlip())}
     return merge_transforms(default_transforms(image_size), transforms)
+
+
+@dataclass
+class ImageClassificationInputTransform(InputTransform):
+
+    image_size: Tuple[int, int] = (196, 196)
+
+    def input_per_sample_transform(self):
+        return T.Compose(
+            [T.ToTensor(), T.Resize(self.image_size), T.Normalize([0.485, 0.456, 0.406], [0.229, 0.224, 0.225])]
+        )
+
+    def train_input_per_sample_transform(self):
+        return T.Compose(
+            [
+                T.ToTensor(),
+                T.Resize(self.image_size),
+                T.Normalize([0.485, 0.456, 0.406], [0.229, 0.224, 0.225]),
+                T.RandomHorizontalFlip(),
+            ]
+        )
+
+    def target_per_sample_transform(self) -> Callable:
+        return torch.as_tensor
+
+    def collate(self) -> Callable:
+        return kornia_collate
```

Comment:
Probably don't need this if we are not using Kornia

Reply:
Some tests are failing if we don't use this one. Let me investigate this deeper.

Reply:
Ok, it would fail for `tests/image/classification/test_training_strategies.py::test_learn2learn_training_strategies` if we use default_collate. I will keep it for now.